CONTRIBUTING.md

```diff
@@ -52,3 +52,8 @@ $> ./vendor/bin/phpcs --standard="vendor/aik099/coding-standard/CodingStandard"
 ```
 
 or by making your IDE ([instructions for PhpStorm](http://www.jetbrains.com/phpstorm/webhelp/using-php-code-sniffer-tool.html)) to check them automatically.
+
+As an aid to code formatting, a `.editorconfig` file is supplied.  If your IDE can support it, (instructions for 
```

Comment:
Here you say, that `.editorconfig` file is supplied, but I don't see it as part of this PR.

Reply:
fixed